File: DemoObjc/DemoObjc/ViewController.m
//
//  ViewController.m
//  DemoObjc
//
//  Created by Fujiki Takeshi on 2018/07/18.
//  Copyright © 2018 takecian. All rights reserved.
//

#import "ViewController.h"
@import SwiftRater;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [SwiftRater checkWithHost:self];
//    [SwiftRater rateApp];
}

@end

File: Demo/Demo/ViewController.swift
//
//  ViewController.swift
//  Demo
//
//  Created by Fujiki Takeshi on 2017/03/28.
//  Copyright © 2017年 com.takecian. All rights reserved.
//

import UIKit
import SwiftRater

class ViewController: UIViewController {

    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
        SwiftRater.incrementSignificantUsageCount(point: 1)
        SwiftRater.check(host: self)

        // If your want to show rating dialog manually, use `rateApp()`.
        // SwiftRater.rateApp(host: self)
    }

}
